tests/suites/test_suite_mpi.function

```diff
@@ -763,6 +763,271 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void mpi_core_cond_assign( data_t * input_X,
+                           data_t * input_Y )
+{
+    #define MAX_LEN 64
+    mbedtls_mpi_uint *X = NULL;
+    mbedtls_mpi_uint *Y = NULL;
+    size_t len_X = CHARS_TO_LIMBS( input_X->len );
+    size_t len_Y = CHARS_TO_LIMBS( input_Y->len );
```

Comment:
Naming: please use `limbs` for sizes in limbs, and reserve `len/length` for sizes in bytes (`bytes` is fine too).